var/spack/repos/builtin/packages/py-zc-lockfile/package.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyZcLockfile(PythonPackage):
+    """Basic inter-process locks """
+
+    homepage = "http://www.python.org/pypi/zc.lockfile"
+    url      = "https://pypi.io/packages/source/z/zc.lockfile/zc.lockfile-1.4.tar.gz"
+
+    version(
+        '1.4', sha256='95a8e3846937ab2991b61703d6e0251d5abb9604e18412e2714e1b90db173253')
+
+    depends_on('py-setuptools', type=('build', 'run'))
+    depends_on('py-zc-buildout', type=('build', 'run'))
```

Comment:
This dependency isn't listed in the `setup.py`, remove it

Reply:
Yes, it's not listed, but It's used in their bootstrap.py...

Reply:
What is this `bootstrap.py` file used for? Is it something that gets installed? Maybe we could make it a `run` type dependency only.

Reply:
Okay, well I could have sworn it wouldn't install unless I added the py-zc-buildout package, but I just installed it from the recipe with the dependency removed and its fine without it.  Sorry for the confusion.